1_prep.R

```diff
@@ -59,9 +50,32 @@ p1 <- list(
     p1_nldas_dates,
     munge_nldas_dates(p1_nldas_csvs, p1_nldas_time_period)),
 
+
+  # Prep observed data for calibration ------------------
+
+  # Pull in observed data from `lake-temperature-model-prep``
+  tar_target(p1_merged_temp_data_daily_feather,
+             '1_prep/in/merged_temp_data_daily.feather',
+             format = 'file'),
+
+  # create an RDS file for each MO model for calibration
+  tar_target(p1_obs_rds,
+             subset_model_obs_data(data = p1_merged_temp_data_daily_feather,
+                                   nhdhr_id = p1_nldas_site_ids,
```

Comment:
The reason I called the variable `nhdhr_id` is because [this](https://github.com/padilla410/lake-temperature-missouri-models/blob/6dcd8f520ad46e3382eb8152fdd79b560b3865ec/1_prep/src/munge_obs.R#L5-L6) line of code doesn't work if it's `dplyr::filter(site_id %in% site_id)`. I think a reasonable compromise would be something like `site_id_filter`. Looks like I also left out the doc.